### PR TITLE
Replaced webL10n with i18next in Calligra activity

### PR DESCRIPTION
**Pull request by anudeeps0306**
Issue #1379 

Replaced webL10n with i18next in Calligra activity

Screenshot from 2024-02-16 16-10-47

### CODE REVIEW

**Comment by llaske** [comment]
The tutorial doesn't work

image

File: activities/Calligra.activity/index.html [hunk 1]
```diff
@@ -5,7 +5,7 @@
 <meta charset="utf-8" />
 <title>Calligra Activity</title>
 <meta name="viewport" content="user-scalable=no, initial-scale=1, maximum-scale=1, minimum-scale=1, width=device-width, viewport-fit=cover"/>
-<link rel="prefetch" type="application/l10n" href="locale.ini" />
+<!-- <link rel="prefetch" type="application/l10n" href="locale.ini" /> -->
```
Comment (llaske) [inline activities/Calligra.activity/index.html:8]: Remove the line, do not comment

Comment (anudeeps0306) [reply activities/Calligra.activity/index.html]: @llaske, I've made some changes, and now everything is functioning correctly locally. However, I'm having trouble understanding the conflicting file.

**Comment by llaske** [comment]
Words are not localized 

image

**Comment by llaske** [comment]
Plus the `locale.ini` should be removed.

**Comment by anudeeps0306** [comment]
@llaske I have corrected the 
Screenshot from 2024-02-18 11-01-30
necessary changes.
Screenshot from 2024-02-18 11-01-49

**Comment by llaske** [comment]
There is a conflict. It's related to missing localization strings from insert words popup.

image

**Comment by anudeeps0306** [comment]
@llaske I have made necessary changes.

**Comment by llaske** [comment]
Nice. Thanks

**Pull request merged**